Docs/system_diagram.mmd

```diff
@@ -0,0 +1,28 @@
+ C4Context
+      title System Context diagram for LLM Benchmark System
+      Boundary(b0, "Dutch Government") {
+        System(SystemLLMBenchmark, "LLM Benchmark System", "Allows users to upload LLM models, upload LLM Benchmarks, and test LLMs against Dutch LLM benchmarks.")
+        System(SystemAlgoReg, "Algorithm Register", "Registers the algorithms in use by the Dutch government")
+      }
+
+      Person(dev0, "Local LLM Developer", "Developer that develops LLMs and tests locally")
+      Person(dev, "LLM Developer", "Developer that develops LLMs.")
+      Person(dev1, "LLM Benchmark Developer", "Developer that develops LLM Benchmarks")
+
+      Rel(dev0, SystemLLMBenchmark, "Uses the system locally")
+      Rel(dev, SystemLLMBenchmark, "Uploads LLM model")
```

Comment:
I'm not sure there will/should be any uploading to this system. I'd rather say something like "refers to available model" and "refers to LLM behind an API"